lib/active_model/serializer/adapter/json/fragment_cache.rb

```diff
@@ -1,16 +1,6 @@
-require 'active_model/serializer/adapter/fragment_cache'
-module ActiveModel
-  class Serializer
-    class Adapter
-      class Json < Adapter
-        class FragmentCache
+class ActiveModel::Serializer::Adapter::Json::FragmentCache
```

Comment:
Re: the namespace issue, I'm going to update the PR description to reference it.  I also discussed it in https://github.com/rails-api/active_model_serializers/pull/1050#issuecomment-133295789 and https://github.com/rails-api/active_model_serializers/pull/1067#discussion_r37710003. The code in this PR was my first attempt to deal with https://github.com/rails-api/active_model_serializers/issues/1006 till I realized a simpler solution..